translations/Polish/research/notes.md

```diff
@@ -0,0 +1,13 @@
+General notes:
+
+* The main difference between Polish and English writing are the special letters, in English, there is no ą, ć, ę, ł, ń, ś, ó, ż, ź.
+* Polish grammar differs much from the English. What matters in the context of websites, texts are usually longer in Polish.
+* Polish and English are both written from left to right.
+
+Thoughts after analysing the screenshots:
+ 
+* Polish words and sentences are quite often longer what forces sites to scale. For instance: Google_preferences, what fitted on one site displayed in English, I had to scroll in Polish.
```

Comment:
I'd say `which forces sites`.

Not sure what you mean by `scale` here, did you mean `adjust`?

`For instance in Enligsh Google_preferences fit above the fold, but I had to scroll the site in Polish`